config/gnu-flags

```diff
@@ -98,7 +98,7 @@ fi
 
 if test "X-gcc" = "X-$cc_vendor"; then
 
-    CFLAGS="$CFLAGS -Wall -Wcast-qual -Wconversion -Wextra -Wfloat-equal -Wformat=2 -Winit-self -Winvalid-pch -Wmissing-include-dirs -Wshadow -Wundef -Wwrite-strings -pedantic -Wno-c++-compat"
+    CFLAGS="$CFLAGS -std=c99 -Wall -Wcast-qual -Wconversion -Wextra -Wfloat-equal -Wformat=2 -Winit-self -Winvalid-pch -Wmissing-include-dirs -Wshadow -Wundef -Wwrite-strings -pedantic -Wno-c++-compat"
```

Comment:
I think it would be best to use the compiler flags files and "add flags" macros from hdf5 so we can be sure that CMake and autotools are using the same flags.

Reply:
Doesn't CMake just do the right thing when you set the standard, though?

Reply:
Yes, CMake seems to be fine.